### PR TITLE
Remove unused code from CSV

### PR DESCRIPTION
**Pull request by nijel**
This is not being used since cb74f6805ac6cd7e6046938b3d5b2088519b2680, so there is probably no sense to keep it around.

### CODE REVIEW

File: translate/storage/csvl10n.py [hunk 1]
```diff
@@ -25,72 +25,9 @@
 import six
 
 from translate.misc import csv_utils
-from translate.misc import sparse
```
Comment (unho) [inline translate/storage/csvl10n.py]: Just verified that `translate/misc/sparse.py` can be also dropped. Please include its removal in this PR.

Comment (nijel) [inline translate/storage/csvl10n.py]: Good point, I've looked only at SimpleParser class which is used in selector, but it's also defined there...

**Comment by unho** [comment]
lgtm once tests pass.

File: translate/storage/csvl10n.py (hunk 1 above)
Comment (unho) [inline translate/storage/csvl10n.py]: Yes, that is a different `SimpleParser` class.

**Pull request merged**